packages/react/package.json

```
@@ -23,12 +23,12 @@
     "url": "https://github.com/vmware/clarity.git"
   },
   "dependencies": {
-    "@lit-labs/react": "^1.0.1",
-    "react": "^16.13.1",
-    "react-dom": "^16.13.1"
+    "@lit-labs/react": "^1.0.1"
   },
   "peerDependencies": {
-    "@cds/core": "./dist/core"
+    "@cds/core": "./dist/core",
+    "react":"^16.13.1 || ^17.0.2",
+    "react-dom": "^16.13.1 || ^17.0.2"
```

It's probably fine to be a little looser for v17:

```suggestion
    "react":"^16.13.1 || 17",
    "react-dom": "^16.13.1 || 17"
```

(Maybe even for v16 but hard to say for sure)

---

@ashleyryan If we don't expect React to change the behaviour of the code we use what do you think about doing what we did for the [@cds/angular](https://github.com/vmware/clarity/blob/4006bea6dfe7b31a4708cc656dc628aad47562fa/packages/angular/projects/cds-angular/package.json#L17)?

We had issues before when a newer version of the Angular was released. The compiler was raising an error that `@cds/angular` expects to work with 3 specific versions of Angular which was not true and we had to relax the version specification. 

I believe that's what @gnomeontherun had in mind with [the comment above](https://github.com/vmware/clarity/pull/6466/files#r751557277)

---

Right, I assumed that's what he was getting at - changing the peer dependency to be >16. They've been making a lot of changes in React 18 and I don't know how that will behave with Clarity, which is why I'm hesitant to add blanket support for it. 

But it's only in Beta so I think it would be safe to say `> 16` with a disclaimer that we don't guarantee React 18 support yet

---

I prefer to keep it less strict unless we know it doesn't work, otherwise we might not get feedback because people won't be able to install it.

---

Makes sense - updated!